common/display/src/main/scala/uk/ac/wellcome/display/models/WorksIncludes.scala

```diff
@@ -29,20 +33,31 @@ object V1WorksIncludes {
 }
 
 object V2WorksIncludes {
+
   val recognisedIncludes = List(
     "identifiers",
     "items",
     "subjects",
     "genres",
     "contributors",
-    "production")
+    "production",
+    "notes",
+    "contents",
+    "credits",
+    "dissertation",
+  )
+
   def apply(includesList: List[String]): V2WorksIncludes = V2WorksIncludes(
     identifiers = includesList.contains("identifiers"),
     items = includesList.contains("items"),
     subjects = includesList.contains("subjects"),
     genres = includesList.contains("genres"),
     contributors = includesList.contains("contributors"),
-    production = includesList.contains("production")
+    production = includesList.contains("production"),
+    notes = includesList.contains("production"),
```

Comment:
Typo?

Reply:
Still working on this one, would be great if you could review #186